boards/lilygo_twatch_2020/axp202c.py

```diff
@@ -0,0 +1,820 @@
+'''

```

Comment:
Is `axp202c` a twach specific configuration?  
It looks more like a driver. Maybe it is also used by some other boards?  
A similar driver (`axp192`) is [already under `driver/generic`](https://github.com/lvgl/lv_binding_micropython/blob/master/driver/generic/axp192.py)